test/unit/org/apache/cassandra/utils/streamhist/StreamingTombstoneHistogramBuilderTest.java

```diff
@@ -138,7 +137,7 @@ public void testNumericTypes() throws Exception
 
         asMap = asMap(deserialized);
         assertEquals(1, deserialized.size());
-        assertEquals(Integer.MAX_VALUE, asMap.get(2).intValue());
+        assertEquals(Integer.MAX_VALUE, asMap.get(2l).intValue());
```

Comment:
Nit: `2L` instead of `2l`, just to make IDEs happy about readability inspections.

Reply:
Will do